spec/db/migration_7_spec.rb

```diff
@@ -0,0 +1,18 @@
+describe "migration 7" do
+  include_context "with database"
+
+  let(:migrator) { Migrator.new }
+
+  before do
+    migrator.migrate_to(database, 6)
+  end
+  it "Update old forms with a created and updated date" do
+
+    @database[:forms].where(created_at: nil).update(
```

Comment:
Wait don't merge, I just noticed I don't explicityly create any test cases. Though there SHOULD be stuff in the db to test, I'd want to add some.